lib/fs.js

```diff
@@ -0,0 +1,32 @@
+const gracefulFs = require('graceful-fs')
+const promises = gracefulFs.promises
+
+/**
+ * Add fs.rm for older versions of node
+ */
+promises.rm = promises.rm || async function (...args) {
```

Comment:
Do we want to mutate `rm` for everyone, or maybe spread and and it just for us in this wrapper file

Reply:
I'm optimizing for smallest amount of changes in other files (as much as I can, this entire endeavor was probably a bit of a bad idea)

Reply:
Oh wait I see what you mean